xonsh/xonfig.py

```diff
@@ -353,6 +353,8 @@ def _info(ns):
         ('is superuser', is_superuser()),
         ('default encoding', DEFAULT_ENCODING),
         ])
+    if githash():
+        data.append(('git SHA', githash()))
```

Comment:
It is probably OK to always have this as part of the `xonfig`. If it is None, we'll see none